### PR TITLE
Seo

### PR DESCRIPTION
**Pull request by maehr**
# Pull request

- [x] #50
- [x] #51
- [x] #52
- [x] #53
- [x] https://crawler.siteone.io/html/2024-01-22/30d/g940-hho2t7-29mznfpy.html

## Types of changes

- [x] New feature (non-breaking change which adds functionality).
- [x] Enhancement (non-breaking change which enhances functionality)
- [x] Bug Fix (non-breaking change which fixes an issue).
- [ ] Breaking change (fix or feature that would cause existing functionality to change).

## Checklist

- [x] I have read the **[README](./README.md)** document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

## Summary by CodeRabbit

- **New Features**
    - Introduced a new link for the apple-touch-icon in the HTML head for better mobile device compatibility.
    - Added dynamic email and Instagram handle configuration.
    - Enhanced SEO and social media sharing capabilities with dynamic metadata and structured data (JSON-LD) in head components across various pages.
    - Improved website navigation and user experience by dynamically adjusting page layouts and content visibility based on URL path.
    - Updated the error page with a more informative layout and integrated analytics.
    - Enhanced blog and archive pages with dynamic SEO metadata.
    - Updated external links and license information in the credits page.
    - Added a Sitemap directive in `robots.txt` for improved search engine indexing.

- **Bug Fixes**
    - Fixed email link dynamics in the footer to use the configured email address.

- **Documentation**
    - Updated documentation links and license information to reflect the correct sources.

- **Tests**
    - Updated navigation tests to reflect changes in routes and expected page titles.

### CODE REVIEW

File: src/routes/agenda/+page.svelte [hunk 1]
```diff
@@ -201,13 +208,11 @@
...
 		<p>
 			Alle Daten auf dieser Seite stammen von <a href="https://agendabasel.ch">agendabasel.ch</a>
-			und werden der Stiftung <a href="{base}/ueber-uns">Stadt.Geschichte.Basel</a> im Rahmen einer
-			Kooperation kostenlos zur Verfügung gestellt. Falls Sie eine Veranstaltung oder Ausstellung
-			auf Stadt.Geschichte.Basel publizieren möchten, wenden Sie sich bitte direkt an agendabasel.ch
-			oder an
-			<a href="mailto:vermittlung@stadtgeschichtebasel.ch">vermittlung@stadtgeschichtebasel.ch</a>.
-			Die Stiftung Stadt.Geschichte.Basel übernimmt keine Verantwortung für die Richtigkeit der
-			Daten.
+			und werden der <a href="{base}/ueber-uns">{config}</a> im Rahmen einer Kooperation kostenlos
```
Comment (koilebeit) [inline src/routes/agenda/+page.svelte:211]: - small error here:
```suggestion
			und werden der <a href="{base}/ueber-uns">{config.title}</a> im Rahmen einer Kooperation kostenlos
```

**Review by koilebeit — COMMENTED** [review]
it seems that the sitemap is not created correctly: at least for me, the <?xml> ddefinition and the xmlns-attribute in the urlset tag are missing.
But maybe I have overlooked something?

**Comment by maehr** [comment]
> it seems that the sitemap is not created correctly: at least for me, the <?xml> ddefinition and the xmlns-attribute in the urlset tag are missing. But maybe I have overlooked something?

I checked the sitemap with https://www.mysitemapgenerator.com/service/check.html and fixed some errors. The xml did not fail. What do you want to add exactly?

**Comment by maehr** [comment]
PS: Hypotheses.org is down https://www.isitdownrightnow.com/sgb.hypotheses.org.html

**Review by koilebeit — APPROVED** [review]
the xml is valid. When I run dev, sitemap.xml begins with 
```
<urlset>
```
instead of 
```
<?xml version="1.0" encoding="UTF-8"?>
<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
```
..but ok 🤷

**Comment by maehr** [comment]
> the xml is valid. When I run dev, sitemap.xml begins with
> 
> ```
> <urlset>
> ```
> 
> instead of
> 
> ```
> <?xml version="1.0" encoding="UTF-8"?>
> <urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
> ```
> 
> ..but ok 🤷

I cannot reproduce this. If I run `pnpm run build` I get

```sitemap.xml
<?xml version="1.0" encoding="UTF-8"?><urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"><url><loc>http://sveltekit-prerender/baende</loc><lastmod>2024-01-15T15:27:24</lastmod><changefreq>weekly</changefreq><priority>1.0</priority></url><url><loc>http://sveltekit-prerender/datenschutzerklaerung</loc><lastmod>2024-01-18T10:51:31</lastmod><changefreq>weekly</changefreq><priority>1.0</priority></url><url><loc>http://sveltekit-prerender/pressemappe</loc><lastmod>2024-01-18T10:45:55</lastmod>
...
```

**Pull request merged**